guidelines/index.html

```diff
@@ -993,9 +993,9 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
  
  <p>例外: テキスト表記においてこれらのテキストスタイルプロパティの一つ以上を使用しない自然言語及び文字体系では、その言語と文字体系の組み合わせに存在するプロパティだけを用いて、この達成基準に適合することができる。</p>
 
- <div class="note" role="note" id="issue-container-generatedID-10"><div role="heading" class="note-title marker" id="h-note-10" aria-level="5"><span>注記 1</span></div><p class="">Content is not required to use these text spacing values. The requirement is to ensure that when a user overrides the authored text spacing, content or functionality is not lost.</p></div>
+ <div class="note" role="note" id="issue-container-generatedID-10"><div role="heading" class="note-title marker" id="h-note-10" aria-level="5"><span>注記 1</span></div><p class="">コンテンツが、これらのテキストの間隔の値を用いる必要はない。要求事項は、制作されたテキストの間隔を利用者が上書きした場合に、コンテンツ又は機能が失われないことである。</p></div>
 
- <div class="note" role="note" id="issue-container-generatedID-11"><div role="heading" class="note-title marker" id="h-note-11" aria-level="5"><span>注記 2</span></div><p class="">Writing systems for some languages use different text spacing settings, such as paragraph start indent. Authors are encouraged to follow locally available guidance for improving readability and legibility of text in their writing system.</p></div>
+ <div class="note" role="note" id="issue-container-generatedID-11"><div role="heading" class="note-title marker" id="h-note-11" aria-level="5"><span>注記 2</span></div><p class="">一部の言語の書記体系では、例えば段落開始のインデントのように、異なるテキストの間隔の設定が用いられているものもある。コンテンツ制作者は、テキストの可読性及び視認性を向上させるために、地域で入手可能なその書記体系のガイドラインに従うことが推奨される。</p></div>
```

Comment:
https://github.com/waic/wcag22/pull/46#discussion_r1435024934 では "Writing systems for some languages use different..." を 「言語の中には、書記体系において、」としていました。

「書記体系で…用いられている」というのはごくわずかに違和感があります。書記体系というのは体系であり、未来に書かれるテキストにも適用されるルールであるはずなので、「書記体系で…用いられる」と現在形にした方が違和感がないです (個人の感覚かも)。

locally available は、ここでは、その地域で入手できるという意味ではなく、その地域の言語に対して適用できるという意味ではないでしょうか。
(言語も書記体系も共通だが、地方で異なる要素がある、という想定?)

このような感じでどうでしょう?

```suggestion
 <div class="note" role="note" id="issue-container-generatedID-11"><div role="heading" class="note-title marker" id="h-note-11" aria-level="5"><span>注記 2</span></div><p class="">言語の中には、書記体系において、例えば段落開始のインデントのように、異なるテキストの間隔の設定が用いられるものもある。コンテンツ制作者は、テキストの可読性及び視認性を向上させるために、その書記体系の、その地域に適用されるガイドラインに従うことが推奨される。</p></div>
```